src/main.ts

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable */
```

Comment:
Why's this necessary here?

Reply:
Eslint will try turning this into:
```
import type moment, { WeekSpec } from 'moment';
```

which isn't valid Typescript.

But let me fix this since there's a better approach outlined here: https://github.com/benmosher/eslint-plugin-import/issues/1869